schematools/contrib/django/models.py

```diff
@@ -133,6 +137,8 @@ class DynamicModel(models.Model):
     _table_schema: DatasetTableSchema = None
     _display_field = None
 
+    objects = Manager()
+
```

Comment:
Waarom is dit nodig?

Reply:
Niet nodig, in een eerste poging had ik een custom manager gemaakt, maar dat bleek later niet geschikt. Deze kan weg.